protocol.md

```diff
@@ -311,20 +320,28 @@ This packet is sent by the client to request channel messages that have not been
 Use `ChannelId=0` to get messages from arbitrary channels.  
 Setting `After` or `Before` to `0` disables the boundry check.
 
-After sending all requested messages, the servers sends a _SyncFinshed_ packet. Because of dependencies it will happen almost every time that the client will receive messages from the server that it already has. These messages have to be ignored.
+Just like the session restore, this action starts a sync process with _SyncStarted_ and _SyncFinshed_ packets.
+It is likely that the client will receive messages from the server that it already has. These messages have to be ignored.
 ```vpsl
 <Int64 ChannelId>
 <Int64 After><Int64 Before>
 <UInt16 MaxCount>
 ```
 
-### **0x0F** SyncFinished ![networkDown] ###
-This packet is sent by the server to a client to indicate that all messages have been delivered. If the client has a _WakeLock_ or something like this to prevent the users device from sleeping, it can be released after receiving this message.
+### **0x0B** SyncStarted ![networkDown] ###
+This packet is sent by the server to a client to indicate that a sync process has started.
+`MinCount` specifies how many channel messages are going to be sent at least. A value of `-1` means that the server did not check the message count in advance. Clients must not consider a sync process to be complete before they receive the respective _SyncFinished_ packet.
```

Comment:
Just out of interest: When and why does the server not check how many messages it is going to send?

Reply:
Counting the number of messages requires an additional query before fetching the message contents from the database. If a working progress bar is not worth the performance impact, this feature could be turned of in the server.